### PR TITLE
Prep for laravel docker-compose. Add Git Submodule.

### PR DESCRIPTION
**Pull request by exrhizo**
Add some structure around how env vars are done.

Add laravel submodule.

vscode 2 space, wasn't defaulting for me.. So put it in the local config, it's our convention

Pin dependencies

### CODE REVIEW

File: src/lib/README [hunk 1]
```diff
@@ -1,2 +1,4 @@
+# libraries
```
Comment (levity) [inline src/lib/README:1]: Rename file to .md if you want to use markdown

File: src/config/env.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  : '10000000-ffff-ffff-ffff-000000000001';
+export const INFURA_PROJECT_ID = getEnvValue(
+  'REACT_APP_INFURA_PROJECT_ID',
+  'missing-infura-id'
+);
+export const REACT_APP_HASURA_URL = getEnvValue(
+  'REACT_APP_HASURA_URL',
+  'https://missing-hasura-url.edu'
+);
+export const HASURA_ADMIN_SECRET = process.env.REACT_APP_HASURA_ADMIN_SECRET;
```
Comment (levity) [inline src/config/env.ts:38]: I think we need to avoid putting this in the frontend build, no?

**Review by levity — CHANGES_REQUESTED** [review]

File: src/lib/README (hunk 1 above)
Comment (exrhizo) [reply src/lib/README:1]: done

File: src/config/env.ts (hunk 2 above)
Comment (exrhizo) [reply src/config/env.ts]: done

**Review by levity — APPROVED** [review]

**Pull request merged**